### ISSUE TITLE
[Bug] Subscriptions Broken

### ISSUE THREAD

**Issue by Abystus** [iv-org/invidious#3123]
**Describe the bug**
Error appears when attempting to load subscriptions using https://vid.puffyan.us/feed/subscriptions.

**Steps to Reproduce**

1. Log in
2. Move to subscriptions tab (https://vid.puffyan.us/feed/subscriptions)

**Screenshots**
![image](https://user-images.githubusercontent.com/1529049/170615750-227058df-b80e-41e6-8485-a5bb805fe8f4.png)

**Comment by Captain-Le-honk** [comment]
I am having the same issue

**Comment by felixdv** [comment]
Same here

**Comment by unixfox** [comment]
Is this still an issue?

**Comment by Abystus** [comment]
@unixfox It seems to be resolved. Thanks!

**Comment by GauthierPLM** [comment]
I had the same issue after upgrading my PG cluster from v12 to v14. Reading at the error, the view existed but was simply missing data, making it invalid.
I connected via psql and ran `REFRESH MATERIALIZED VIEW <view name>;` and it fixed the issue.

@unixfox I believe the solution would be for Invidious to refresh the view soon after starting or if this error is caught.

**Comment by unixfox** [comment]
> I had the same issue after upgrading my PG cluster from v12 to v14. Reading at the error, the view existed but was simply missing data, making it invalid.
> I connected via psql and ran `REFRESH MATERIALIZED VIEW <view name>;` and it fixed the issue.
> 
> @unixfox I believe the solution would be for Invidious to refresh the view soon after starting or if this error is caught.

The real solution is to merge this PR: https://github.com/iv-org/invidious/pull/2469
But it requires some DB changes.

**Comment by Quix0r** [comment]
I also came across this one and the temporary solution was tor recreate the views. After that, the granted permissions were gone and an error message about this came. I then granted access to these individual views (I only have 2 users, including myself). I still feel that this might not be the best, flexible solution, but the subscriptions view works again.
`GRANT SELECT, INSERT, UPDATE, DELETE ON subscriptions_<ID_HERE> TO kemal;`
